settings/common.py

```diff
@@ -45,6 +45,7 @@
 ]
 
 OUR_APPS = [
+    'evalai',
```

Comment:
@TroJan yeah, I knew this would arise and yeah, I too think `base` app would be a good location for `management` directory.